cmd/spec/example_pubkey.go

```diff
@@ -29,4 +32,8 @@ var PubkeyListResponse = payloads.PubkeyListResponse{
 			FingerprintLegacy: "ee:f1:d4:62:99:ab:17:d9:3b:00:66:62:32:b2:55:9e",
 		},
 	},
+	Metadata: &page.Metadata{
+		Previous: "/api/provisioning/v1/pubkeys?limit=100&offset=0",
+		Next:     "/api/provisioning/v1/pubkeys?limit=100&offset=2",
```

Comment:
This is not correct, right? the offset should be `100`, or limit `2` right? :thinking: 
Having examples correct goes long way of explaining the funcionality :)

Reply:
It is correct if you have only two pubkeys :) 
What would you expect to see if you have seen the last page?

Reply:
Maybe it is more correct to put the limit (100)

Reply:
Ah, in that case that goes to the discussion if we should have next empty, as it basically does not make sense to have next, that will give you empty result IMO :thinking: